src/api/app/views/webui2/webui/repositories/_repository_entry.html.haml

```diff
@@ -0,0 +1,7 @@
+.col-12.col-md-6.col-lg-4
+  .card.mb-3.repository-card
+    .card-header
+      = link_to(repository, project_repository_state_path(project: project, repository: repository), class: 'font-weight-bold')
```

Comment:
The repository's name is missing in the path, so the path should be:

project_repository_state_path(project: project, repository: repository<b>.name</b>)

Otherwise the path is generated with the ID when it needs the name.